bude_hezky/request/request_maker.py

```diff
@@ -0,0 +1,21 @@
+import requests
+from requests.exceptions import HTTPError
+
+SUCCESSFUL_STATUS_CODE_THRESHOLD = 299;
+
+class RequestException(Exception):
+    pass
+
+def make_request(url, params, response=None):
+    try:
+        response = response or requests.get(url, params=params)
+        response.raise_for_status()
+    except HTTPError as http_err:
+        raise RequestException(f'HTTP error occurred: {http_err}') from http_err
```

Comment:
Chápu, že si to chceš obalit, ale asi by se dalo spolehnout i na to, že `requests` vyhodí svoji `requests.exceptions.RequestException()`

Info:

- https://requests.kennethreitz.org/en/master/user/quickstart/#errors-and-exceptions

Reply:
(Tenhle soubor ve výsledku může obsahovat tak čtyři řádky.)